### DISCUSSION TITLE
Dashboard refresh rate

### DISCUSSION THREAD

**Discussion by JSJ10FL** [kimai/kimai#3435] [Questions & Answers]
I am very new to this, so please bear with me if there is an easy answer. I have worked on a project for 9:30 hours, and that shows up in the reporting page, and in 'my times', but not yet in the Dashboard. Does the Dashboard only refresh at the end of day? Or have I missed something? Thank you.

**Comment by kevinpapst** [comment]
Kimai does not cache these informations, everything is displayed real time.
Sure you looked at the correct week? Share some screenshots please

**Comment by JSJ10FL** [comment]
Thank you for coming back to me

Screen shot showing 9.30 hours worked in the ‘Strategic Plan’ project
Screen shot showing only 1hr in same project on Dashboard
It’s all in the same week, I’ve only been using it for a week!
